### PR TITLE
Fix close linkages with new workflow.

### PR DESCRIPTION
**Pull request by gosso22**

### CODE REVIEW

File: app/src/main/java/org/smartregister/addo/interactor/AddoVisitInteractor.java [hunk 1]
```diff
@@ -186,6 +186,10 @@ protected void submitVisit(boolean editMode, String memberID, Map<String, BaseAn
             }
         }
 
+        JSONObject dangerSignsForm = new JSONObject(dangerSignJsonString);
```
Comment (maulidkategile) [inline app/src/main/java/org/smartregister/addo/interactor/AddoVisitInteractor.java]: @gosso22 This needs to be handled when dangerSignJsonString is empty for the use case when a client has a prescription note

**Review by maulidkategile — COMMENTED** [review]
@gosso22 Is the Botnar logo retained on the sidebar of the home page?

File: app/src/main/java/org/smartregister/addo/interactor/AddoVisitInteractor.java (hunk 1 above)
Comment (gosso22) [reply app/src/main/java/org/smartregister/addo/interactor/AddoVisitInteractor.java]: Thanks for this feedback Maulid, it helped to identify a problem from this logic, when creating a followup tasks for linkage  we need to use details from the original linkage i.e. focus and referral problems. So we dont need to use the danger signs screening form here.

**Review by issyzac — APPROVED** [review]
Looking good!

**Pull request merged**